server/controllers/products.js

```diff
@@ -25,15 +47,14 @@ module.exports = (req, res, next) => {
 	res.vary('referer');
 	res.vary('user-agent');
 
-	module.exports.getProducts({
+	getProducts({
```

Comment:
I find this a bit messy to read, especially with the trailing countryCode param. 
Could we extract the ammitParams to a separate const and then do `getProducts(ammitParams, countryCode)`?
It also makes the code more meaningful, you know right away what the first param is without looking at the function prototype.